### PR TITLE
Fix the install instructions for Typescript core

### PR DESCRIPTION
**Pull request by payoto**
Based on our chat on discord, feel free to close if you're going to do it yourself 😄

### CODE REVIEW

**Comment by sestinj** [comment]
Looks perfect, thanks for beating me to it! Only small change is "nmp" -> "npm" in install-dependencies.sh

File: install-dependencies.sh [hunk 1]
```diff
@@ -3,11 +3,23 @@
...
+echo "Installing Core extension dependencies..."
+pushd core
+npm install
+nmp link
+popd
 
+echo "Installing GUI extension dependencies..."
+pushd gui
+npm install
+nmp link core
```
Comment (payoto) [inline install-dependencies.sh]:
```suggestion
npm link core
```

File: install-dependencies.sh [hunk 2]
```diff
@@ -3,11 +3,23 @@
...
+popd
 # VSCode Extension (will also package GUI)
 echo "Installing VSCode extension dependencies..."
-pushd extensions/vscode || exit
+pushd extensions/vscode
 
 # This does way too many things inline but is the common denominator between many of the scripts
-npm install || exit
+npm install
+nmp link core
```
Comment (payoto) [inline install-dependencies.sh]:
```suggestion
npm link core
```

File: install-dependencies.sh [hunk 3]
```diff
@@ -3,11 +3,23 @@
 # Start developing with:
 # - Run Task -> Install Dependencies
 # - Debug -> Extension
+set -e
+echo "Installing Core extension dependencies..."
+pushd core
+npm install
+nmp link
```
Comment (payoto) [inline install-dependencies.sh]:
```suggestion
npm link
```

**Comment by payoto** [comment]
Silly mistake 🤦 no problem, happy to help

**Comment by sestinj** [comment]
Thanks so much!

**Pull request merged**